src/main/java/Duke.java

```diff
@@ -1,10 +1,260 @@
+
+	// max no. of tasks in taskList
+	private static final int MAX_TASKS = 100;
+	// array of class Task
+	private static final Task[] taskList = new Task[MAX_TASKS];
+	;
+	private static int listLength = 0;
+
+	private static final String LINE_PARTITION = "    ____________________________________________________________\n";
+	private static final String FAREWELL = LINE_PARTITION +
+			"\t  Bye. Hope to see you again soon!\n" +
+			LINE_PARTITION;
+	private static final String GREET = LINE_PARTITION +
+			"\t  Hello! I'm kimo\n" +
+			"\t  What can I do for you?\n" +
+			LINE_PARTITION;
+	private static final String TASK_ADD_SUCCESS_MSG = LINE_PARTITION +
+			String.format("\t  added:%s\n" +
+					"\t  You now have %d task(s) in your list.\n", taskList[listLength - 1], listLength) +
+			LINE_PARTITION;
+
+
+	//ERROR MESSAGE FINAL STRINGS
+	private static final String ERROR_FACE = "(ಠ_ಠ)";
+	private static final String OUT_OF_BOUNDS_MSG = String.format(LINE_PARTITION +
+			"\t  There are %d items in your list.\n", listLength) +
+			"\t  Please enter a number in the appropriate range." + ERROR_FACE + "\n" +
+			LINE_PARTITION;
+	private static final String INVALID_ARGS_MSG = LINE_PARTITION +
+			"\t  You did not enter a number.\n" +
+			"\t  Try again." + ERROR_FACE + "\n" +
+			LINE_PARTITION;
+	private static final String CMD_FORMAT_ERROR = LINE_PARTITION +
+			"\t  You have entered the wrong format for this command.\n" +
+			"\t  Please type \"help\" to see the list of commands and the relevant formats " + ERROR_FACE + "\n" +
+			LINE_PARTITION;
+	private static final String INVALID_CMD = LINE_PARTITION +
+			"\t  You have entered an invalid command.\n" +
+			"\t  Please type \"help\" to see the available list of commands and the relevant formats\n" +
+			LINE_PARTITION;
+
+
+	//HELP MESSAGE
+	private static final String HELPLIST = LINE_PARTITION +
+			"\t  These are the valid commands and the relevant formats for each:\n" +
+			"\t  list : displays the current tasklist of tasks you have entered, with each tasks' statuses shown." +
+			"\t  mark (no.) : marks task number (no.) as completed.\n" +
+			"\t  unmark (no.) : marks task number (no.) as incomplete.\n" +
+			"\t  todo (task) : adds task with description (task) to the tasklist.\n" +
+			"\t  deadline (task) /by (date) : adds task with description (task) and due date (date) to the tasklist" +
+			".\n" +
+			"\t  event (task) /from (date1) /to (date2) : adds task with description (task) " +
+			"and start date (date1) and end date (date2)\n" +
+			"\t  help : shows list of available commands\n" +
+			LINE_PARTITION;
+
+	//MAIN CODE
+	public static void main(String[] args) {
+		System.out.print(GREET);
+		while (true) {
+			takeUserInput();
+		}
+	}
+
+	public static void takeUserInput() {
+		final String userInput = in.nextLine();
+		final String[] cmdTypeAndParams = splitUserInput(userInput);
+		final String cmd = cmdTypeAndParams[0];
+		final String args = cmdTypeAndParams[1];
```

Comment:
Maybe these variables are not required to be `final`? Or variables with `final` should be CAPITALISED.